app/controllers/tasks_controller.rb

```diff
@@ -8,7 +8,7 @@ class TasksController < ApplicationController
     erb :"tasks/new.html"
   end
 
-  post '/tasks' do
+  post '/tasks/create' do
```

Comment:
Typically, we wouldn't add `'/create'` here, as that is implied by the fact that it is a `POST` request